release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2720,6 +2720,78 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+[id="ocp-4-15-12-pipeline-list-page-improvement"]
+===== PipelineRun list view performance improvement
+
+* Previously, in the PipelineRun list page, all of the TaskRuns were fetched and separated based on their PipelineRun name. With this release, TaskRuns are only fetched for failed and cancelled PipelineRuns, and a caching mechanism is added to fetch the PipelineRuns and TaskRuns that are associated with the failed and cancelled PipelineRuns. (link:https://issues.redhat.com/browse/OCPBUGS-31799[*OCPBUGS-31799*])
+
+[id="ocp-4-15-12-installer-handling-new-character"]
+===== Installer handles the escaping of the "%" character
+
+* Previously, if a cluster was installed using proxy and the proxy information contained escaped characters in the format "%XX", the installation failed. With this release, the Installer now handles the escaping of the "%"" character. (link:https://issues.redhat.com/browse/OCPBUGS-32259[*OCPBUGS-32259*])
+
+[id="ocp-4-15-12-cfe-evaluation-status"]
+===== Cluster Fleet Evaluation status information added to the Machine Config Operator
+
+* Previously, the Machine Config Operator (MCO) did not include the Cluster Fleet Evaluation (CFE) Status. With this release, the CFE status information is added to the MCO and available to customers. (link:https://issues.redhat.com/browse/OCPBUGS-32922[*OCPBUGS-32922*])
+
+[id="ocp-4-15-12-operatorhub-filter-rename-FIPS"]
+===== OperatorHub filter renamed from "FIPS Mode" to "Designed for FIPS"
+
+* Previously, OperatorHub included a filter named "FIPS Mode". With this release, that filter is named "Designed for FIPS". (link:https://issues.redhat.com/browse/OCPBUGS-32933[*OCPBUGS-32933*])
+
+[id="ocp-4-15-12-bug-fixes"]
+==== Bug fixes
+
+* Previously, containers had an incorrect view of the pids limit in their `cgroup` hierarchy and reported as a random number instead of `max`. The containers do not have max pids and are limited by the pod pid limit, which is set outside of the container's `cgroup` hierarchy and not visible from within the container. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-28926[*OCPBUGS-28926*])
+
+* Previously, for {product-title} deployments on {rh-openstack-first}, the `MachineSet` object did not correctly apply the value for the `Port Security` parameter. With this release, the `MachineSet` object applies the `port_security_enabled` flag as expected. (link:https://issues.redhat.com/browse/OCPBUGS-30857[*OCPBUGS-30857*])
+
+* Previously, the Installer erroneously attempted to verify the libvirt network interfaces when an agent-based installation was configured with the openshift-baremetal-install binary. With this release, the agent installation method does not require libvirt and this validation is disabled. (link:https://issues.redhat.com/browse/OCPBUGS-30944[*OCPBUGS-30944*])
```

Comment:
Similar thought re: "installer" but if you know from doing release notes a bunch that I'm wrong on that, please ignore. 🙃 
```suggestion
* Previously, the installer erroneously attempted to verify the libvirt network interfaces when an agent-based installation was configured with the openshift-baremetal-install binary. With this release, the agent installation method does not require libvirt and this validation is disabled. (link:https://issues.redhat.com/browse/OCPBUGS-30944[*OCPBUGS-30944*])
```